assets/maps/prefabs/prefab_dans_asteroid.dmm

```diff
@@ -0,0 +1,3173 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
This map is still TGM formatted. Prefabs must use the BYOND DMM format

Reply:
Should be .dmm in BYOND format now. :)